common-tools/clas-detector/src/main/java/org/jlab/detector/scalers/DaqScalers.java

```diff
@@ -95,11 +99,13 @@ public static double getSeconds(Date rst,Date uet) {
     public static DaqScalers create(Bank runScalerBank) {
         DaqScalers ds=new DaqScalers();
         ds.dsc2=new Dsc2Scaler();
-        for (int ii=0; ii<runScalerBank.getRows(); ii++) {
-            ds.dsc2.setLivetime(runScalerBank.getFloat("livetime", ii));
-            ds.dsc2.setBeamCharge(runScalerBank.getFloat("fcup",ii));
-            ds.dsc2.setBeamChargeGated(runScalerBank.getFloat("fcupgated",ii));
-            break; 
+        if (runScalerBank.getRows() == 1) {
+            ds.dsc2.setLivetime(runScalerBank.getFloat("livetime", 0));
+            ds.dsc2.setBeamCharge(runScalerBank.getFloat("fcup",0));
+            ds.dsc2.setBeamChargeGated(runScalerBank.getFloat("fcupgated",0));
+        }
+        else if (runScalerBank.getRows() > 1) {
+            logger.warning("found event where RUN::scaler bank has more than 1 row");
```

Comment:
Let's make sure `ds.dsc2` members are still set when rows > 1.
```suggestion
        if (runScalerBank.getRows() >= 1) {
            ds.dsc2.setLivetime(runScalerBank.getFloat("livetime", 0));
            ds.dsc2.setBeamCharge(runScalerBank.getFloat("fcup",0));
            ds.dsc2.setBeamChargeGated(runScalerBank.getFloat("fcupgated",0));
        }
        if (runScalerBank.getRows() > 1) {
            logger.warning("found event where RUN::scaler bank has more than 1 row");
```